Automattic/wp-calypso
# Plugins update manager: Schedule form validation

Related to https://github.com/Automattic/dotcom-forge/issues/5544


## Proposed Changes

* Prepared schedule form for the BE integration
* Added form validation covered with unit tests
  * plugins: doesn't allow the creation of the two same sets of plugins for update
  * timeslot: doesn't allow the same timeslot for scheduling updates

## Testing Instructions

* Go to `/plugins/update-manager/create/{SITE_SLUG}`
* Check the name and plugins selection validation
* The rest "special cases", we need backend integration; check the unit tests

![Screen Capture on 2024-02-28 at 12-03-33](https://github.com/Automattic/wp-calypso/assets/1241413/7d08b7e2-05f9-40f2-b634-2c43e35c7a4b)


## Pre-merge Checklist
